subprojects/common/graphite/src/main/kotlin/com/avito/android/graphite/GraphiteSender.kt

```diff
@@ -13,10 +13,10 @@ interface GraphiteSender {
 
     class Impl(
         private val config: GraphiteConfig,
-        private val logger: Logger
+        loggerFactory: LoggerFactory
     ) : GraphiteSender {
 
-        private val tag = "graphite"
+        private val logger = loggerFactory.create("GraphiteSender")
```

Comment:
It would be convenient to infer a name from a class

```kotlin
private val logger = loggerFactory.create<GraphiteSender>()
```
It looks like a common scenario.

Reply:
done, and added to doc